### DISCUSSION TITLE
Allow Geofences to be Grouped

### DISCUSSION THREAD

**Discussion by Starmixcraft** [teslamate-org/teslamate#1476] [Ideas]
My suggestion is to allow geo-fences to be grouped into Fencgroups, therefor you can add multiple geo-fences to 1 group and not have them displayed individually in Grafana but as one location. This is very useful if you have 2 Apartments and want to mark them both at Home but not want 2 Different Homes to show up on you're Stats.

**Comment by adriankumpf** [comment]
Moving this to [💡 Ideas](https://github.com/adriankumpf/teslamate/discussions/categories/ideas)
